tests/typing/callable.py

```diff
@@ -28,24 +30,34 @@ def create(cls) -> Animal:
 
 # Test 4: to check the .args & .kwargs attributes
 provider4 = providers.Callable(Animal)
-args4: Tuple[Any] = provider4.args
-kwargs4: Dict[str, Any] = provider4.kwargs
+args4 = provider4.args
+kwargs4 = provider4.kwargs
+assert_type(args4, Tuple[Any])
+# TODO: Change Callable.kwargs to Dict[str, Any]? Then adjust test back to Dict[str, Any]
+assert_type(kwargs4, Dict[Any, Any])
 
 # Test 5: to check the provided instance interface
 provider5 = providers.Callable(Animal)
-provided5: Animal = provider5.provided()
-attr_getter5: providers.AttributeGetter = provider5.provided.attr
-item_getter5: providers.ItemGetter = provider5.provided["item"]
-method_caller: providers.MethodCaller = provider5.provided.method.call(123, arg=324)
+provided_val5 = provider5.provided()
+attr_getter5 = provider5.provided.attr
+item_getter5 = provider5.provided["item"]
+method_caller5 = provider5.provided.method.call(123, arg=324)
+# TODO: Remove explicit typing of Provider.provided return type
```

Comment:
The stub is this

```python
    @property
    def provided(self) -> ProvidedInstance[T]: ...
```

But` ProvidedInstance` is not declared to be generic

```python
class ProvidedInstance(Provider, ProvidedInstanceFluentInterface):
    def __init__(self, provides: Optional[Provider] = None) -> None: ...
```

I think we can make the quick fix to
```python
    @property
    def provided(self) -> ProvidedInstance: ...
```